x-pack/legacy/plugins/siem/public/components/alerts_viewer/translations.ts

```diff
@@ -14,8 +14,8 @@ export const TOTAL_COUNT_OF_ALERTS = i18n.translate('xpack.siem.alertsView.total
   defaultMessage: 'alerts match the search criteria',
 });
 
-export const ALERTS_TABLE_TITLE = i18n.translate('xpack.siem.alertsView.alertsDocumentType', {
-  defaultMessage: 'Alerts',
+export const ALERTS_TABLE_TITLE = i18n.translate('xpack.siem.alertsView.alertsTableType', {
+  defaultMessage: 'Alert detection frequency',
```

Comment:
it appears this should still be `Alerts` per the screenshot below:

![alerts-table](https://user-images.githubusercontent.com/4459398/72644431-3cd39700-392e-11ea-93af-ad5b2c932c82.png)